sqlmesh/cli/main.py

```diff
@@ -96,7 +96,7 @@ def cli(
 
 
 @cli.command("init")
-@click.argument("sql_dialect")
+@click.argument("sql_dialect", required=False)
```

Comment:
Why is it not required?

Reply:
So that a dbt project can be inited. Later Trey does a check and raises an exception if not specified and not a dbt project.